src/bin/cargo-miri.rs

```diff
@@ -333,10 +333,11 @@ path = "lib.rs"
         None => true,
         Some(target) => target == rustc_version::version_meta().unwrap().host,
     };
-    let sysroot = if is_host { dir.join("HOST") } else { PathBuf::from(dir) };
+    let sysroot = if is_host { dir.join("HOST") } else { PathBuf::from(dir) }; 
```

Comment:
Is that a `\n` turning into a `\r\n`?

Reply:
Yes, I believe so.

Reply:
Then please fix the lineendings. We want `\n` everywhere.

Reply:
Oh, I apologize for the confusion.. it was not a `\n` turning into a `\r\n`, but was an additional space added at the end of the line. I will get rid of the extraneous space!